content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImportWaitForVm.class/__methods__/wait_for_imported_vm.rb

```diff
@@ -1,36 +1,42 @@
+#
+# Description: This method checks if the VM has been successfully imported
+#
+
 module ManageIQ
   module Automate
-    module System
-      module Event
-        module EmsEvent
-          module RHEVM
-            class UpdateVmImportStatus
```

Comment:
@pkliczewski Do you know if the VM will show up in our inventory before the import is complete? The reason I ask is, if it's not included in our inventory until the import is complete, we don't need the additional state machine because the import state machine will catch the VM when it's complete.